### DISCUSSION TITLE
What  should the App Name be?

### DISCUSSION THREAD

**Discussion by gitaaron** [functionland/fx-fotos#124] [General]
1 - Photos
2 - Pics
3 - Futos + 1
4 -  Fotos + 9
5 - Pix
6 - Pixland +1
7 - FulaPix + 1
8 - FulaFotos +1
9 - PhotoZ +1

4 - Fotos - Aaron
7 - FulaPix - Jonathan

**Comment by gitaaron** [comment]
I like 'Fotos' because:

1. Fotos is short
2. It tells people we are like the other boring 'Photos' apps except we are the cool new kid on the block
3. F = Functionland

**Comment by ehsan6sha** [comment]
I like Fotos too
1- unique
2- short/one word and easy to pronounce
3- F is also the start of both Functionland and Fula
4- tells them we are a photos app like what you already know and not something unfamiliar. Kinda the "same UX" message that we want to communicate

**Comment by gitaaron** [comment]
Moein says

> I choose “Futos” for the name of the app, because it is unique, so close to the word “photos”, and I like the two first words, because they represent Functionland for all.
> Fu —> Functionland 👍🏽

**Reply by gitaaron** [reply]
Fu could also mean 'for us'
